AppCenterDataStorage/AppCenterDataStorage/Internal/Client/MSTokenExchange.m

```diff
@@ -6,6 +6,7 @@
 #import "MSAuthTokenContext.h"
 #import "MSConstants+Internal.h"
 #import "MSDataStorageConstants.h"
+#import "MSDataStore.h"
```

Comment:
We have "MSDataStoreInternal.h" which includes this header, too. I think this include is not necessary here